x-pack/plugins/observability_solution/observability/public/pages/alert_details/alert_details.tsx

```diff
@@ -260,12 +264,11 @@ export function getScreenDescription(alertDetail: AlertData) {
 
   The alert details are:
   ${Object.entries(alertDetail.formatted.fields)
-    .map(([key, value]) => `${key}: ${value}`)
+    .map(([key, value]) => `${key}: ${JSON.stringify(value)}`)
```

Comment:
How many tokens is this? I think we generally need to be considerate of how many tokens we send over - both in terms of cost and accuracy (based the assumption that more information = more opportunity for confusion).

Reply:
It's around 600 tokens (varies between depending on the alert). I can cherry-pick specific fields which will bring it down under 400 tokens.

Over time, do you think this will be less of a problem, as the context window increases, and the ability for LLMs to ignore noise?

We still have to be mindful of costs but that's a tradeoff we can make.

Reply:
I think that's fine! Might be good to leave a comment here. I do have some concerns around it being hard to track how many tokens of screen context is going to be sent over, but let's see how it progresses.